cmu_graphics/cmu_graphics.py

```diff
@@ -214,6 +214,17 @@ def cleanAndClose():
     shape_logic.cleanSoundProcesses()
     os._exit(0)
 
+def _safeMethod(appMethod):
+    def m(*args, **kwargs):
+        app = args[0]
+        try:
+            return appMethod(*args, **kwargs)
+        except Exception as e:
+            sys.excepthook(*sys.exc_info())
+            app._errored = True
```

Comment:
Seems like it might simplify the code to set app.stopped here instead of app._errored.

Reply:
Good point -- since you can't un-stop an app

Reply:
One side effect of this is that it displays the inspector. It's a little ugly 

<img width="408" alt="image" src="https://user-images.githubusercontent.com/5986436/195166677-0aad810a-5d39-4dd8-8583-88c8533c6bf3.png">